privacyidea_pam.py

```diff
@@ -122,6 +169,45 @@ def authenticate(self, password):
 
         return rval
 
+    def challenge_response(self, transaction_id, challenge):
+        rval = self.pamh.PAM_SYSTEM_ERR
+
+        message = self.pamh.Message(self.pamh.PAM_PROMPT_ECHO_OFF, challenge)
+        response = self.pamh.conversation(message)
+        chal_response = json.loads(response.resp)
```

Comment:
weird -- my team have it working on Mac with libu2f-host from homebrew, on linux mint with libu2f-host from git and on ubuntu with u2f-host from the PPA.

If you send a challenge manually to your u2f-host command, does it output some text prior to the response?